Makefile

```diff
@@ -1,27 +1,87 @@
-.PHONY: all policy-controller docker-image clean
+.PHONY: all binary test clean help docker-image
+default: help
 
-SRCDIR=.
+# Makefile configuration options 
 CONTAINER_NAME=calico/kube-policy-controller
-
-default: all
-all: policy-controller
-
-# Build the calico/kube-policy-controller Docker container.
+PACKAGE_NAME?=github.com/projectcalico/k8s-policy
+GO_BUILD_VER:=latest
+CALICO_BUILD?=calico/go-build:$(GO_BUILD_VER)
+LDFLAGS=-ldflags "-X $(PACKAGE_NAME)/calicoctl/commands.VERSION=$(CALICOCTL_VERSION) \
```

Comment:
is this copy/paste from calicoctl/Makefile? If so, don't you need to define $CALICOCTL_VERSION, $CALICOCTL_BUILD_DATE and $CALICOCTL_GIT_REVISION? At the least, you'll want to make sure they're properly initialized.